test/init.go

```diff
@@ -62,3 +67,32 @@ func Initialize() {
 	}
 	util.InitializeTemplates(jt, cjt)
 }
+
+// envInit sets up environmental variables for eunomia tests
+func envInit() {
+	logf.Log.Info("Setting up environmental variables")
+	_, initFilename, _, _ := runtime.Caller(0)
+	eunomiaRoot := filepath.Join(filepath.Dir(initFilename), "..")
+	envs := map[string]string{
+		"JOB_TEMPLATE":     filepath.Join(eunomiaRoot, "./build/job-templates/job.yaml"),
+		"CRONJOB_TEMPLATE": filepath.Join(eunomiaRoot, "./build/job-templates/cronjob.yaml"),
```

Comment:
Wouldn't it be easier to pass the paths as regular arguments to the InitializeTemplates function on l.68? Is there some important reason why we must pass them through environment instead?